bus-mapping/src/evm/opcodes/returndatasize.rs

```diff
@@ -26,6 +25,13 @@ impl Opcode for Returndatasize {
             value,
         );
 
+        if value.as_usize() != state.call_ctx()?.return_data.len() {
```

Comment:
This catches an inconsistency in deposit_ether.json, but it doesn't causes any of the circuits to fail. I think I'm looking into why there's a failure.